django_x509/tests/test_ca.py

```diff
@@ -681,3 +681,9 @@ def test_ca_common_name_length(self):
         message_dict = context_manager.exception.message_dict
         self.assertIn('common_name', message_dict)
         self.assertEqual(message_dict['common_name'][0], msg)
+
+    def test_ca_without_key_length_and_digest_algo(self):
```

Comment:
I think this test is wrong. It should test for empty string, not None, which at DB level translates to NULL, which is not accepted at DB level because we don't have `null=True` in the model definition
If I run this test without the patch, I get this error `django.db.utils.IntegrityError: NOT NULL constraint failed: django_x509_ca.key_length`.

Reply:
Thank you for catching this @nemesisdesign. I wiill be more careful from next time.